versioned_docs/version-2.x/magic-button/_category_.yml

```diff
@@ -0,0 +1,10 @@
+label: Magic Button
+position: 91
+className: new
+link:
+  type: generated-index
+  description:
+   Enhance your content creation process with the Magic Button,
```

Comment:
Note: at the moment, the Magic button must added by integrators so that it
appears. It's not documented because we have not decided yet (https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2045#note_81490) if we want to keep it that way or not.